python/llm/src/bigdl/llm/serving/README.md

```diff
@@ -78,7 +78,27 @@ If you run successfully using `BigDL` backend, you can see the output in log lik
 INFO - Converting the current model to sym_int4 format......
 ```
 
-> note: We currently only support int4 quantization.
+> note: We currently only support int4 quantization for this method.
```

Comment:
Shall we simply remove the previous method?

Reply:
Yes, we can do that.  The reason we currently not doing so is that some of the docker images may rely on this method.

We can gradually remove this method in later PRs after the new method has been integrated into bigdl-llm release version.